website/docs/r/wafregional_rule.html.markdown

```diff
@@ -0,0 +1,64 @@
+---
+layout: "aws"
+page_title: "AWS: wafregional_rule"
+sidebar_current: "docs-aws-resource-wafregional-rule"
+description: |-
+  Provides a AWS WAF Regional rule resource for use with ALB.
+---
+
+# aws\_wafregional\_rule
+
+Provides a WAF Regional Rule Resource for use with Application Load Balancer.
+
+## Example Usage
+
+```hcl
+resource "aws_wafregional_ipset" "ipset" {
+  name = "tfIPSet"
+  
+  ip_set_descriptors {
+    type  = "IPV4"
+    value = "192.0.7.0/24"
+  }
+}
+
+resource "aws_wafregional_rule" "wafrule" {
+  depends_on  = ["aws_wafregional_ipset.ipset"]
```

Comment:
This is redundant because the relationship is already built by the reference `${aws_wafregional_ipset.ipset.id}` below. Do you mind removing it?